test/src/xquery/xquery3/higher-order.xml

```diff
@@ -380,15 +380,15 @@ return
         </code>
         <expected>70</expected>
     </test>
-    <test output="text">
+    <test output="text" id="lookup-unknown">
         <task>fn:function-lookup on unknown function</task>
         <code><![CDATA[xquery version "3.0";
 
 declare namespace ex="http://exist-db.org/xquery/ex";
 
 let $f1 := function-lookup(xs:QName("ex:add"), 2)
 return
-    if ($f1) then
+    if (exists($f1)) then
```

Comment:
Hmm. Just wondering why the exists call is needed?

Reply:
Because effectiveBooleanValue is not defined on function references, so the error eXist did throw for this test was correct. I checked against the spec and other implementations.